Sofa/framework/LinearAlgebra/src/sofa/linearalgebra/CompressedRowSparseMatrixGeneric.h

```diff
@@ -34,6 +34,30 @@
 #include <sofa/linearalgebra/matrix_bloc_traits.h>
 #include <algorithm>
 
+namespace // anonymous
```

Comment:
Maybe you could move this code in the `trait` folder

Reply:
in this case, it was of a boiler-plate code to identify the nature of a Block, and its name does not really follow the implicit convention like is_vector, etc...